istioctl/cmd/metics-query.go

```diff
@@ -0,0 +1,138 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/prometheus/prometheus/model/labels"
+)
+
+type Labels []string
+
+type LabelsBuilder struct {
```

Comment:
do prometheus have same struct for this?

Reply:
Promethus do not have the same struct, this structure named possible ambiguity.

like QueryVariableBuilder（Temporarily didn't think of a more appropriate name）

Responsible for the structural query variable， example : sum(rate(istio_request_total{%s}[%s])) by (%s)
Doing so will help keep the query more flexible。